### DISCUSSION TITLE
Point to location for git cli

### DISCUSSION THREAD

**Discussion by georgibakken** [semantic-release/semantic-release#1887] [Q&A]
Hello,

Is there a way to set the path for the installed git cli version that semantic-release will use? 

semantic-release requires 2.7.1, but my CI machine (jenkins) is running with an older verison. My admins however installed a newer version in `/usr/local/bin/git` and not the system wide `/bin/git` which is still running the older version. I want to point semantic-release to use the version located in `/usr/local/bin/git`. Is this possible?

Thank you for any help or suggestions!

**Comment by travi** [comment]
semantic-release will use the git executable that is in the `$PATH`. it should be possible to set this as part of your command, along the lines of `PATH=<value here> npx semantic-release`

**Reply by georgibakken** [reply]
great! thank you for the quick reply 👍
